OWNERS

```diff
@@ -20,11 +20,11 @@
       owners: [{name: 'mrjoro'}, {name: 'ampproject/wg-outreach'}],
     },
     {
-      pattern: '{.*,babel.config.js,gulpfile.js}',
+      pattern: '{.*,gulpfile.js}',
       owners: [{name: 'ampproject/wg-infra'}],
     },
     {
-      pattern: '{package.json,yarn.lock}',
+      pattern: '{package.json,yarn.lock,babel.config.js}',
```

Comment:
Updating this to reflect the ownership of `build.conf.js`, which included runtime and performance.